imap_processing/lo/l1a/lo_data_container.py

```diff
@@ -0,0 +1,66 @@
+"""Container for IMAP-Lo L0 data classes."""
+
+from imap_processing.lo.l0.lo_apid import LoAPID
+
+
+# TODO: Not sure if this class is worth having.
```

Comment:
I wonder if you can simplify this function using functional programming. It's ok if you like to do update in future PR. I know it's hard when you don't have good data to try with.

Reply:
Thanks. I wrote this class months ago thinking it'd be more useful than it is turning out to be. I'll think about it some more.